src/features/MoviesListPage/MainContent/index.js

```diff
@@ -38,13 +39,17 @@ export const MainContent = () => {
 
     const moviesToDisplay = isSearching ? searchResults : popularMovies.results;
 
+    if (isSearching && (!moviesToDisplay || moviesToDisplay.length === 0)) {
+        return <NoResults query={searchQuery} />;
```

Comment:
Good job, but I have a question!
Is it necessary to write both of this requirements in statemant?
**(!moviesToDisplay || moviesToDisplay.length === 0))**
Maybe just !moviesToDisplay or moviesToDisplay.length === 0) will be enough?

Reply:
You're right, using just !moviesToDisplay or moviesToDisplay.length === 0 is a cleaner way to check if the array is empty.
I'll update the code to use your suggested condition.

Reply:
The condition is necessary to display the <NoResults> component with the image. Without this check, the <NoResults> component would render correctly, but the image would not be shown, providing an incomplete user experience.